src/Symfony/Bridge/Doctrine/Messenger/DoctrinePingConnectionMiddleware.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Messenger;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Middleware\MiddlewareInterface;
+use Symfony\Component\Messenger\Middleware\StackInterface;
+
+class DoctrinePingConnectionMiddleware implements MiddlewareInterface
+{
+    /** @var ManagerRegistry */
+    private $managerRegistry;
+
+    /** @var string */
+    private $entityManagerName;
+
+    /** @var bool */
```

Comment:
please remove docblocks, we don't use them when they copy the signature of the constructor

Reply:
Done